test/verify/check-shell-multi-os

```diff
@@ -100,6 +100,9 @@ class TestMultiOS(MachineCase):
         dev_m = self.machine
         dev_b = self.browser
 
+        if self.machine.image == "arch":
+            self.write_file("/etc/hosts", "10.111.113.5 stock", append=True)
```

Comment:
Why doesn't the common function work here? I suppose "stock" is set dynamically somewhere?